### PR TITLE
Import Linux Kernel typos: letter C

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1663,8 +1663,10 @@ bussiness->business
 buttosn->buttons
 butttons->buttons
 buuild->build
+cace->cache
 cach->catch, cache,
 cacheing->caching
+cachlines->cachelines
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: not sure about this one

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -1823,12 +1841,14 @@ cartilege->cartilage
...
 cartrige->cartridge
 cas->case
+cascace->cascade
 case-insenstive->case-insensitive
 case-sensitve->case-sensitive
 caseinsenstive->case-insensitive
 casesensitve->case-sensitive
 casette->cassette
 cashe->cache
+cashes->caches
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: `cashes` is a valid word, i'd add it as a result JIC

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -1875,6 +1896,7 @@ ceated->created
 ceates->creates
 ceating->creating
 ceation->creation
+ceding->cedeing
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: ceding is a correct word

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -1939,14 +1962,18 @@ Champange->Champagne
 chaned->changed, chained,
 chaneged->changed
 chanel->channel
+chanels->channels
+chang->change
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: i bet will get a lot of false positives with this one

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -1980,12 +2007,14 @@ charcter->character
...
 charctor->character
 charctors->characters
+charecters->characters
 charicter->character
 charicters->characters
 charictor->character
 charictors->characters
 chariman->chairman
 charistics->characteristics
+chasnge->change
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: possibly `changes` as well

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -2102,7 +2139,9 @@ classe->class, classes,
 classess->classes
 classs->class
 classses->classes
+clcoksource->clocksource
 clcosed->closed
+cleancache->cleancacne
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: This looks like it should be the other way around

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: added

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Fixed. Thanks.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -2761,12 +2868,18 @@ contibutes->contributes
 contibutor->contributor
 contigent->contingent
 contigious->contiguous
+contigiously->contiguously
+contignuous->contiguous
 contigous->contiguous
 contiguious->contiguous
 contiguos->contiguous
+contiguous-non->non-contigous
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -2983,6 +3109,7 @@ cotrol->control
 cotrolled->controlled
 cotrolling->controlling
 cotrols->controls
+couarpoint->cougarpoint
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -3037,8 +3167,10 @@ crasher->crash
 crashers->crashes
 crashs->crashes
 crated->created, crated,
+crates->creates
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

**Comment by Gelma** [comment]
I'm going to close this and open a new PR.

**Pull request closed without merging**